components/Players/PlayerManager.vue

```diff
@@ -566,6 +572,13 @@ export default Vue.extend({
           ]
         });
       }
+    },
+    toggleFullScreen(): void {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
```suggestion
      // @ts-expect-error
```

Add the relevant description of the reasoning. We might not need this, see below.